src/frontend/src/App.js

```diff
@@ -8,30 +8,45 @@ import {LoginPage} from "./pages/LoginPage";
 import {ProfilePage} from "./pages/ProfilePage";
 import {CreatePage} from "./pages/CreatePage";
 import {SearchPage} from "./pages/SearchPage";
+<<<<<<< HEAD
+import Footer from "./components/footer";
+import { Switch } from "@mui/material";
+import { createContext, useState } from "react";
+export const ThemeContext = createContext(null);
+
+
+=======
 import Footer from "./components/Footer";
 import { ToastContainer, toast } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
+>>>>>>> 0e4db4609b5b994e912d40f97c04ced782ed0ec8
```

Comment:
These are appearing due to merge conflicts, you need to fix this part of the code by removing the unnecessary lines and the repetitive imports

Reply:
Also this part of code is obvious to lead to compilation error but you have still pushed the code, I would recommend you to **ALWAYS** build and run the application locally and confirm that it's working fine before committing and pushing

Reply:
I had run the program locally and then only pushed the code, somehow it had compiled successfully. But I will make the necessary changes to remove the conflict.